### PR TITLE
chore(cli): write tpl and params file for aurora and rdws

### PR DESCRIPTION
**Pull request by efekarakus**
Manual testing results in the resource creations.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: internal/pkg/term/color/color.go [hunk 1]
```diff
@@ -87,8 +87,8 @@ func HighlightCode(s string) string {
 	return HiCyan.Sprintf("`%s`", s)
 }
 
-// HiglightCodeBlock wraps the string s with ``` characters, colors it to denote it's a multi-line code block, and returns it.
-func HiglightCodeBlock(s string) string {
```
Comment (iamhopaul123) [inline internal/pkg/term/color/color.go:91]: 🤣

File: internal/pkg/cli/storage_init.go [hunk 2]
```diff
@@ -718,6 +763,22 @@ func (o *initStorageOpts) environmentNames() ([]string, error) {
...
+	mft, err := o.ws.ReadWorkloadManifest(o.workloadName)
+	if err != nil {
+		return "", fmt.Errorf("read manifest for %s: %w", o.workloadName, err)
+	}
+	t, err := mft.WorkloadType()
+	if err != nil {
+		return "", fmt.Errorf("read 'type' from manifest for %s: %w", o.workloadName, err)
+	}
+	o.workloadType = t
+	return o.workloadType, nil
```
Comment (iamhopaul123) [inline internal/pkg/cli/storage_init.go]: We don't need to return the type but just use `o.workloadType` right? or we abandon `o.workloadType` this cached variable completely

**Review by iamhopaul123 — APPROVED** [review]
LGTM just a very nit comment

**Review by huanjani — APPROVED** [review]
Looks good to me. Added DNM label so you can address PH's Q.

File: internal/pkg/cli/storage_init.go (hunk 2 above)
Comment (efekarakus) [reply internal/pkg/cli/storage_init.go]: Refactored! from now we'll call `readWorkloadType` first in `Execute` and only refer to the cached variable afterwards

**Pull request merged**